modules/tinymce/src/plugins/imagetools/main/ts/ui/Buttons.ts

```diff
@@ -5,55 +5,63 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Toolbar } from '@ephox/bridge';
```

Comment:
This is using the wrong type. It should be using the `Toolbar` type from `tinymce/core/api/ui/Ui` instead.